docker/bin/docker_build.sh

```diff
@@ -40,7 +40,7 @@ DOCKER_IMAGE_TAG="${DOCKER_REPO}/bedrock_${DOCKERFILE}:${DOCKER_TAG}"
 
 # generate the dockerfile
 rm -f "$FINAL_DOCKERFILE"
-cat "docker/dockerfiles/bedrock_$DOCKERFILE" | envsubst '$GIT_COMMIT' > "$FINAL_DOCKERFILE"
+sed -e "s/\${GIT_COMMIT}/${GIT_COMMIT}/g" "docker/dockerfiles/bedrock_$DOCKERFILE" > "$FINAL_DOCKERFILE"
```

Comment:
tip, not relevant to the Circle v2 change:

Docker now supports `--build-arg=` so we could change  the Dockerfile to 

```
ARG GIT_COMMIT
FROM mozorg/bedrock_base:${GIT_COMMIT}
...
```

and forget about env variable substitution!

Reply:
Nice! Thanks! I'll check that out.